R/SIMrun.R

```diff
@@ -1032,7 +1035,7 @@ SIMrun <- function(poppar, limits = NULL, model, data, split,
       }
       predTimes <- sapply(predInt, function(x) rep(seq(x[1], x[2], x[3]), each = numeqt))
       # catenate columns into single vector
-      predTimes <- c(predTimes)
+      predTimes <- unlist(predTimes)
```

Comment:
This is the specific line that changed, other modificattions are just formatting

Reply:
We'd have to test it, but the code change seems logical and sensical.